### PR TITLE
Histogram name fix

### PR DESCRIPTION
**Pull request by lilyclements**
This is a pretty minor edit: I added a underscore between frequency and polynomial in the output name to be consistent with other dialogs.

@africanmathsinitiative/developers this is ready to review

### CODE REVIEW

**Comment by dannyparsons** [comment]
It doesn't look like there's any designer changes?

**Comment by lilyclements** [comment]
I realised I was confusing myself between designer changes here and on another branch (bar chart) so I removed that part of my comment. Sorry!

**Comment by lilyclements** [comment]
I should add in bUserTyped though. I'll do that now.

File: instat/dlgHistogram.vb [hunk 1]
```diff
@@ -238,7 +238,7 @@ Public Class dlgHistogram
         End If
     End Sub
 
-    Private Sub ControlsValueChanged() Handles ucrVariablesAsFactorforHist.ControlContentsChanged, ucrSaveHist.ControlContentsChanged
+    Private Sub CoreControls_ValueChanged() Handles ucrVariablesAsFactorforHist.ControlContentsChanged, ucrSaveHist.ControlContentsChanged
```
Comment (shadrackkibet) [inline instat/dlgHistogram.vb]: should this be CoreControls_ValueChanged() or CoreControls_ContentChanged()

Comment (lilyclements) [reply instat/dlgHistogram.vb]: Thanks, I'll fix this now!

**Comment by shadrackkibet** [comment]
@dannyparsons this is ready for you.

**Pull request merged**